src/sub/tests/conftest.py

```diff
@@ -56,46 +61,71 @@ def get_file(filename, path=THIS_PATH, **overrides):
 
 def pytest_configure():
     """Called before testing begins"""
-    global ddb_process
     for name in ("boto3", "botocore"):
         logging.getLogger(name).setLevel(logging.CRITICAL)
-    if os.getenv("AWS_LOCAL_DYNAMODB") is None:
-        os.environ["AWS_LOCAL_DYNAMODB"] = f"http://127.0.0.1:{CFG.DYNALITE_PORT}"
 
-    # Latest boto3 now wants fake credentials around, so here we are.
     os.environ["AWS_ACCESS_KEY_ID"] = "fake"
     os.environ["AWS_SECRET_ACCESS_KEY"] = "fake"
     os.environ["USER_TABLE"] = "users-testing"
     os.environ["DELETED_USER_TABLE"] = "deleted-users-testing"
     os.environ["ALLOWED_ORIGIN_SYSTEMS"] = "Test_system,Test_System,Test_System1"
     sys._called_from_test = True
 
-    # Locate absolute path of dynalite
-    dynalite = f"{CFG.REPO_ROOT}/node_modules/.bin/dynalite"
 
-    cmd = f"{dynalite} --port {CFG.DYNALITE_PORT}"
-    ddb_process = subprocess.Popen(
-        cmd, shell=True, env=os.environ, stdout=subprocess.PIPE
+def get_free_tcp_port():
+    import socket
+
+    tcp = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    tcp.bind(("", 0))
+    addr, port = tcp.getsockname()
+    tcp.close()
+    return port
+
+
+def _docker_client():
```

Comment:
what is the benefit of this function?

Reply:
It is called in multiple spots so it was a way to enforce the DRY principle.  I can remove if it is bothersome.

Reply:
yeah generally im pissed if i go look for a function def and it is this trivial

Reply:
That is much akin to my feelings about the 0 value docstings in the cfg.py file.